examples/gateway/mlflow_serving/README.md

```diff
@@ -0,0 +1,181 @@
+# Guide to using an MLflow served model with the MLflow AI Gateway
+In order to utilize the MLflow AI Gateway with the MLflow Model serving service, a few steps must be taken 
+in addition to those for configuring access to SaaS models (such as Anthropic and OpenAI). The first and most obvious 
+step that must be taken prior to interfacing with an Mlflow served model is that a model needs to be logged to the 
+MLflow tracking server. 
+
+An important consideration for deciding whether to interface the MLflow AI Gateway with a specific model is to evaluate the PyFunc interface that the model will 
+return after being called for inference. Due to the fact that the AI Gateway defines a specific response signature, expectations for each route type's payload contents 
+must be met in order for a route to be valid. 
+
+For example, an embeddings route (llm/v1/embeddings route type) is designed to return embeddings data as a collection (a list) of floats that correspond to each of the 
+input strings that are sent for embeddings inference to a service. The expectation that the embeddings route definition has is that the data is in a particular format. Specifically one that 
+is capable of having the embeddings data extractable from a service response. Therefore, an MLflow model that returns data in the format below is perfectly valid. 
+
+```json
+{"predictions": [
+  [0.0, 0.1], 
+  [1.0, 0.0]
+]
+}
```

Comment:
Why not format it  ? It is displayed like
<img width="234" alt="image" src="https://github.com/mlflow/mlflow/assets/19235986/c874dd07-94f9-4168-8bed-314a7a5bdfe4">